pkg/dock/proto/dock.proto

```diff
@@ -366,6 +368,8 @@ message EnableReplicationOpts {
     string context = 15;
     // The replication metadata
     map<string, string> metadata = 16;
+    // Whether is primary replication
+    bool  isPrimary = 17;
```

Comment:
I‘m a bit confused if it's ok to add another field here, is it better to pass ti inside `metadata`?